OpenTabletDriver.UI.Library/Styles/DefaultDark/Controls/RangedInput.axaml

```diff
@@ -0,0 +1,51 @@
+<Styles
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:tc="using:OpenTabletDriver.UI.TemplatedControls">
+  <Style Selector="tc|RangedInput">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Border
+            Name="PART_Background"
+            Classes="Container">
+          <Grid>
+            <Grid.ColumnDefinitions>
+              <ColumnDefinition
+                  Width="Auto"
+                  SharedSizeGroup="Input_LabelColumn" />
+              <ColumnDefinition Width="*" />
+              <ColumnDefinition Width="6*" />
```

Comment:
what does this do?

Reply:
It takes up 6 times more space than other columns with `*`. Is used to scale slider.